go.sum

```diff
@@ -73,6 +73,8 @@ github.com/PuerkitoBio/purell v1.1.0/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbt
 github.com/PuerkitoBio/purell v1.1.1/go.mod h1:c11w/QuzBsJSee3cPx9rAFu61PvFxuPbtSwDGJws/X0=
 github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
 github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578/go.mod h1:uGdkoq3SwY9Y+13GIhn11/XLaGBb4BfwItxLd5jeuXE=
+github.com/RiRa12621/openshift-route-monitor-operator v0.0.0-20200827095438-67fbb79f804a h1:8FFLSGAapF8JdDdSoiulCLGAwGc1X54yEHdWicJYqKA=
```

Comment:
Are we sure that'll work in the future?

Reply:
I will `go mod tidy` so they will be deleted